shell.nix

```diff
@@ -0,0 +1,54 @@
+{
+  githubPAT ? builtins.getEnv "GITHUB_PAT",
+}:
+let
+  nixpkgs = fetchTarball "https://github.com/NixOS/nixpkgs/tarball/nixos-24.11";
+  pkgs = import nixpkgs {
+    config = { };
+    overlays = [ ];
+  };
+
+  bky-as = pkgs.buildGoModule rec {
+    pname = "bky-as";
+    version = "v0.1.0-beta.4";
+
+    tmpDir = "/tmp";
+
+    env = {
+      GOPRIVATE = "github.com/blocky/*";
+      HOME = tmpDir;
+    };
+
+    src = builtins.fetchGit {
+      rev = "f6a2c0f965cfca9583ab29e10b9e2e5acf006046";
+      url = "git@github.com:blocky/delphi.git";
+    };
+
+    doCheck = false;
+
+    preBuild = ''
+      echo machine github.com login doesNotMatter password ${githubPAT} > ${tmpDir}/.netrc
+    '';
+
+    postBuild = ''
+      alias bky-as=cli
+    '';
+
+    vendorHash = "sha256-GXlZz3L5vd1v9NHlaagKw6aY3LEyt9E10reh6EvZ4Bw=";
+
+  };
+in
+pkgs.mkShellNoCC {
+  packages = [
+    pkgs.git
+    pkgs.go
+    pkgs.gotools # for tools like goimports
+    pkgs.golangci-lint # for linting go files
+    pkgs.tinygo # for building wasm
```

Comment:
do we need to pin `go` or `tinygo` to a version?

Reply:
the versions that is pinned by the version that was part of nixpkgs 24.11.  Which is go 1.23.6 and tinygo 0.34.   If we want to spend a few more cycles on it we may be able to tighten up those versions, but in the short term, my main goal is to figure out how to get versions of bky-as to aligh with the project.